Cards/MadGraph5_aMCatNLO/QCD/QCD0B-4Jets-BPSFilter_HT-1000to1200_madgraphMLM-pythia8/QCD0B-4Jets-BPSFilter_HT-1000to1200_madgraphMLM-pythia8.json

```diff
@@ -0,0 +1,20 @@
+{
+    "gridpack_submit": false,
+    "gridpack_path" : "QCD/QCD-4Jets_HT-1000to1200_madgraphMLM-pythia8",
+    "fragment": ["Generator/ExternalLHEProducer_MadGraph5_aMCatNLO.dat", "PartonShower/madgraphMLM-pythia8-hepmcfilter.dat", "Filter/LHE_B_EQ0_filter.dat"],
+    "fragment_user": [],
+    "fragment_vars": {
+        "processParameters": [
+            "'JetMatching:qCut = 14.'",
+            "'JetMatching:nQmatch = 5'",
+            "'JetMatching:nJetMax = 4'",
+            "'TimeShower:mMaxGamma = 4.0'"
+        ],
+        "nAttemptsFilter"  : ["22"],
+        "hepmcFilter" : [
+            "cms.string('PartonShowerBsHepMCFilter')",
+            "filterParameters = cms.PSet()" 
```

Comment:
This one is empty and not used but could take arguments ?

Reply:
For this particular case, yes it don't take any arguments